ui/src/views/CubeProjectNew.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <side-pane :is-open="true" :title="title" @close="onCancel">
+    <form @submit.prevent="onSubmit">
+      <b-message v-if="error" type="is-danger">
+        {{ error }}
+      </b-message>
+
+      <b-field label="Project name">
+        <b-input v-model="project.label" />
+      </b-field>
+
+      <b-field label="Start project from:" :message="projectTypeMessage">
+        <b-radio-button
+          v-for="option in projectSourceKinds"
+          :key="option.value"
+          v-model="project.projectSourceKind"
+          :native-value="option.value"
+        >
+          {{ option.label }}
+        </b-radio-button>
+      </b-field>
+
+      <b-field>
+        <button type="submit" class="button is-primary">
+          Create project
+        </button>
+        <b-button @click="onCancel">
+          Cancel
+        </b-button>
+      </b-field>
+    </form>
+  </side-pane>
+</template>
+
+<script>
+import Vue from 'vue'
+import SidePane from '@/components/SidePane.vue'
+
+export default Vue.extend({
+  name: 'CubeProjectNew',
+  components: { SidePane },
+
+  data () {
+    return {
+      project: {
+        projectSourceKind: 'CSV',
+        label: '',
+      },
+      error: null,
+      projectSourceKinds: [
+        { value: 'CSV', label: 'CSV file(s)' },
+        { value: 'Existing Cube', label: 'Existing cube' },
+      ],
+    }
+  },
+
+  computed: {
+    operation () {
+      return this.$parent.projectsCollection.actions.create
+    },
```

Comment:
Maybe it would be cleaner to make the `operation` a component `prop` and pass in form the parent?

Reply:
I did it this way because `CubeProjectNew` is a separate route. I'm not sure you can pass props down from parent route to child route (to be verified)

Reply:
If not from parent then maybe from `$state`?

Reply:
done